hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/RSProcedureDispatcher.java

```diff
@@ -471,9 +476,10 @@ public ServerOperation(RemoteProcedure remoteProcedure, long procId, Class<?> rs
       this.rsProcData = rsProcData;
     }
 
-    public RemoteProcedureRequest buildRequest() {
+    public RemoteProcedureRequest buildRequestWithMasterStartCode(long masterStartCode) {
```

Comment:
Do not need to change the method I suppose.

Reply:
Method name ? or signture ? 

I considered it while writing: should I add the masterStartCode directly to the RemoteOperation class or pass it later? However, there's a significant asymmetry in the code. The way we build the RegionOpenOperation, RegionCloseOperation, and ServerOperation differs substantially from each other.

Reply:
I think @Apache9 is referring to not required to change Method name.

Reply:
Ah yes, exactly as @ranganathg said.